set.mm

```diff
@@ -63368,10 +63368,10 @@ from the cartesian product of two singletons onto a singleton (case where
     $( A function's value belongs to the image.  (Contributed by Glauco
        Siliprandi, 23-Oct-2021.) $)
     fnfvimad $p |- ( ph -> ( F ` B ) e. ( F " C ) ) $=
-      ( cin cima cfv wss inss2 imass2 ax-mp a1i wfun cdm wa wcel fnfun sseqtr4d
-      wfn syl inss1 fndmd jca elind funfvima2 sylc sseldd ) AEBDIZJZEDJZCEKZUMU
-      NLZAULDLUPBDMULDENOPAEQZULERZLZSCULTUOUMTAUQUSAEBUCUQFBEUAUDAULBURULBLABD
-      UEPABEFUFUBUGABDCGHUHULCEUIUJUK $.
+      ( cin cima cfv wss inss2 imass2 ax-mp wfun cdm wa wcel wfn fnfun syl sylc
+      inss1 fndmd syl5sseqr jca elind funfvima2 sseldi ) AEBDIZJZEDJZCEKZUKDLUL
+      UMLBDMUKDENOAEPZUKEQZLZRCUKSUNULSAUOUQAEBTUOFBEUAUBABUKUPBDUDABEFUEUFUGAB
+      DCGHUHUKCEUIUCUJ $.
```

Comment:
The proof of this statement given at fnfvima2 is apparently shorter. It should probably be put here, with a further `MINIMIZE_WITH *` just in case.